build/AzurePipelinesTemplates/ProjectReunion-CreateNugetPackage-Job.yml

```diff
@@ -54,8 +58,41 @@ jobs:
 
   - task: DownloadBuildArtifacts@0 
     inputs: 
-      artifactName: drop
-      downloadPath: '$(Build.SourcesDirectory)\Artifacts'
+      artifactName: fatnuget
+      downloadPath: '$(Build.ArtifactStagingDirectory)'
+
+  - task: CopyFiles@2
+    displayName: 'copy license to fat nuget'
+    inputs:
+      SourceFolder: '$(Build.SourcesDirectory)'
+      Contents: |
+        LICENSE
+      TargetFolder: '${{ parameters.fatnupkgdir }}'
+
+  # Extract thin nuget package from fat nuget package
+  - task: CopyFiles@2
+    displayName: 'Extract Thin Nuget from Fat Nuget'
+    inputs:
+      SourceFolder: '${{ parameters.fatnupkgdir }}'
+      Contents: |
+        build\**
+        include\**
+        lib\**
+        LICENSE
+      TargetFolder: '${{ parameters.thinnupkgdir }}'
+      flattenFolders: false
+
+  # Extract framework package from fat nuget package
+  - task: CopyFiles@2
+    displayName: 'Extract Framework Package from Fat Nuget'
+    inputs:
+      SourceFolder: '${{ parameters.fatnupkgdir }}'
+      Contents: |
+        runtimes\**
+        lib\uap10.0\**
```

Comment:
>uap10.0 [](start = 12, length = 7)

also need net5.0-windows folder

Reply:
The net5.0 stuff goes into both the AppX and the thin nuget package?  That doesn't seem correct.  I actually didn't even want to put the winmds in here originally, since it's included in the thin nuget package.

Reply:
I assume you either include both or neither.

---
In reply to: [511201967](https://github.com/microsoft/ProjectReunion/pull/252#discussion_r511201967) [](ancestors = 511201967)